### PR TITLE
2.8.3 release preparations

### PR DESCRIPTION
**Pull request by richardc**
Bump the version and update the release notes for 2.8.3

### CODE REVIEW

File: website/releasenotes.md [hunk 1]
```diff
@@ -8,6 +8,57 @@ This is a list of release notes for various releases, you should review these
...
+application.
+
+Some exmples of this might be, find nodes with the apache configuraion
+class:
+
+{% highlight shell %}
+$ mco describe_filter -C apache
+-C filter expands to the following class checks:
+
+  Check if class 'moon' is present on the host
```
Comment (ploubser) [inline website/releasenotes.md]: Class Apache?

File: website/releasenotes.md [hunk 2]
```diff
@@ -8,6 +8,57 @@ This is a list of release notes for various releases, you should review these
...
+class:
+
+{% highlight shell %}
+$ mco describe_filter -C apache
+-C filter expands to the following class checks:
+
+  Check if class 'apache' is present on the host
+{% endhighlight %}
+
+Find apache nodes on the moon:
```
Comment (ploubser) [inline website/releasenotes.md]: Find Wizard

Comment (richardc) [inline website/releasenotes.md]: Correcting with:

Find hosts with the wizard class that came from the moon:

Comment (ploubser) [inline website/releasenotes.md]: :+1:

**Pull request merged**